css/main.css

```diff
@@ -105,8 +105,8 @@ body {
 .container {
     width: min(99%, 90.5rem);
     margin-inline: auto;
-    overflow-x: scroll;
-    min-height: 85%;
+    overflow-x: auto;
+    min-height: 95vh;
```

Comment:
This seems to have a side effect of pushing the horizontal scrollbar too low that it's not visible at all.

For accessibility reasons, it's desirable to keep the scrollbar visible.